RuntimeLib/StellarMastery.h

```diff
@@ -3,12 +3,32 @@
 #include "Base.h"
 #include "Constants.h"
 
+
 EXTERN_C_BEGIN
 
 #pragma pack(push, 1)
 
+enum {
+    RUNTIME_STELLAR_MASTERY_SLOT_LINK_GRADE_YEARNING = 1,
+    RUNTIME_STELLAR_MASTERY_SLOT_LINK_GRADE_SORROW = 2,
+    RUNTIME_STELLAR_MASTERY_SLOT_LINK_GRADE_WRATH = 3,
+    RUNTIME_STELLAR_MASTERY_SLOT_LINK_GRADE_OBLIVION = 4,
+    RUNTIME_STELLAR_MASTERY_SLOT_LINK_GRADE_VOID = 5,
+};
+
+enum {
+    RUNTIME_STELLAR_MASTERY_FORCE_VALUE_TYPE_MULTIPLICATIVE = 0,
+    RUNTIME_STELLAR_MASTERY_FORCE_VALUE_TYPE_ADDITIVE = 1,
+};
+
 struct _RTStellarMasterySlot {
-    UInt8 Data[16];
+    UInt8 GroupID;
+    UInt8 SlotLine;
+    UInt8 SlotIndex;
+    UInt8 StellarLinkGrade;
```

Comment:
You could remove the prefix `Stellar` from the names as the type already says `RTStellarMasterySlot`